R/zchunk_L210.water.demand.primary.R

```diff
@@ -1,20 +1,21 @@
 #' module_water_L210.water.demand.primary
 #'
-#' Briefly describe what this chunk does.
+#' This chunck is to generate (water demand) technology coefficient for each GCAM region for base years and future years.
```

Comment:
"Generate (water demand) technology coefficients for each GCAM region for base years and future years."